gather_istio

```diff
@@ -68,6 +68,9 @@ function getCRDs() {
     result+=" ${crd} "
   done
 
+  # FIXME: Add a proper query for these CRD's below once https://issues.jboss.org/browse/MAISTRA-799 is fixed.
```

Comment:
Would it make sense to use something like this?

> oc get crds --all-namespaces | grep 'istio.io\|maistra.io'

Reply:
It works too, but I'm not sure if this is the best way. Perhaps a matter of personal preference? I prefer to query things using labels, thus we wouldn't rely on names following a certain pattern.

Reply:
Yeah, I'm not sure what's better but a hardcoded list is much harder to maintain than including any resources from Maistra. We will have to be diligent in making sure we remove this.

Reply:
Querying for maistra.io CRDs and excluding the ones already labelled as part of the Service Mesh would be a safer alternative

Reply:
Done. I'm now using a combination of labels and grep to get the remaining CRD's.